howto/deploy-k8s-prod/README.md

```diff
@@ -0,0 +1,187 @@
+  issuer.key: <ISSUER-KEY-VALUE>
+kind: Secret
+```
+
+Copy the contents of `ca.crt`, `issuer.crt` and `issuer.key` and base64 decode them.
+
+Finally, you should have the following files saved on your disk:, containing the base64 decoded text from the secret:
+
+1. ca.crt
+2. issuer.crt
+3. issuer.key
+
+#### Updating the control plane pods
+
+Next, we need to find a Helm Chart version that installs our desired version of Dapr and perform a `helm upgrade` operation.
+
+First, update the Helm Chart repos:
+
+```bash
+helm repo update
+```
+
+List all charts in the Dapr repo:
+
+```bash
+helm search repo dapr
+
+NAME     	CHART VERSION	APP VERSION	DESCRIPTION
+dapr/dapr	0.4.3        	0.9.0      	A Helm chart for Dapr on Kubernetes
+dapr/dapr	0.4.2        	0.8.0      	A Helm chart for Dapr on Kubernetes
+```
+
+The APP VERSION column tells us which Dapr runtime version is installed by the chart.
+
+Use the following command to upgrade Dapr to your desired version:
+
+```bash
+helm upgrade dapr dapr/dapr --version <CHART-VERSION> --namespace dapr-system --reset-values --set-file dapr_sentry.tls.root.certPEM=ca.crt --set-file dapr_sentry.tls.issuer.certPEM=issuer.crt --set-file dapr_sentry.tls.issuer.keyPEM=issuer.key
+```
+
+Kubernetes will now issue a rolling update.
+Wait until all the new pods appear as running:
+
+```bash
+kubectl get po -n dapr-system -w
+
+NAME                                     READY   STATUS    RESTARTS   AGE
+dapr-dashboard-dcd7cc8fb-ml2p8           1/1     Running   0          12s
+dapr-operator-7b57d884cd-6spcq           1/1     Running   0          12s
+dapr-placement-86b76f6545-vkc6f          1/1     Running   0          12s
+dapr-sentry-7c7bf75d8b-dcnhm             1/1     Running   0          12s
+dapr-sidecar-injector-7b847db96f-kwst9   1/1     Running   0          12s
+```
+
+You can verify the health and version of the control plane using the Dapr CLI:
+
+```bash
+dapr status -k
+
+NAME                   NAMESPACE      HEALTHY  STATUS   VERSION  AGE  CREATED
+dapr-placement         dapr-system    True     Running  0.9.0    12s  2020-07-24 15:38.15
+dapr-operator          dapr-system    True     Running  0.9.0    12s  2020-07-24 15:38.15
+dapr-sidecar-injector  dapr-system    True     Running  0.9.0    12s  2020-07-24 15:38.16
+dapr-sentry            dapr-system    True     Running  0.9.0    12s  2020-07-24 15:38.15
+```
+
+*Note: If new fields have been added to the target Helm Chart being upgraded to, the `helm upgrade` command will fail. If that happens, you will need to find which new fields have been added in the new chart and add them as parameters to the upgrade command, for example: `--set <field-name>=<value>`.*
+
+#### Updating the data plane (sidecars)
+
+The last step is to update pods that are running Dapr to pick up the new version of the Dapr runtime.
+To do that, simply issue a rollout restart command for any deployment that has the `dapr.io/enabled` annotation:
```

Comment:
How can you get a list of all the deployments that have dapr.io/enabled? It would be useful to provide this deployment query here to help get the list of Dapr enabled deployments

Reply:
Kubernetes doesn't have a way to query deployments with specific annotations, but we provide this functionality for users with the Dapr CLI and the Dapr Dashboard. I added an explanation on how to do that.